server/index.js

```diff
@@ -0,0 +1,63 @@
+const path = require('path');
+const express = require('express');
+const request = require('request');
+
+const app = express();
+
+app.use('/', express.static(path.join(__dirname, '../client/dist')));
+app.use('/buildings/:workspaceId', express.static(path.join(__dirname, '../client/dist')));
+
+app.get('/api/availability', async (req, res) => {
```

Comment:
Is there a point of making these async functions?  the node `request` module uses a callback.  Also, curious why you decided to use `request` over a different library.  I heard they actually deprecated this module early 2020.